### PR TITLE
merge citest branch to dltsdev to sync

### PR DESCRIPTION
**Pull request by YinYangOfDao**

### CODE REVIEW

**Comment by xudifsd** [comment]
Oh, please rebase to dltsdev whenever dltsdev branch get updated when you're developing in another branch, do not just merge dltsdev, it will make your PR ugly(contains a lot of Merge commits) like this one.

**Comment by xudifsd** [comment]
I think this will create a real k8s cluster, right? jobmanager will need a real k8s cluster to have some unit tests, do not need a gpu worker though. How can I add that in this CI pipeline?

**Comment by hongzhili** [comment]
> Oh, please rebase to dltsdev whenever dltsdev branch get updated when you're developing in another branch, do not just merge dltsdev, it will make your PR ugly(contains a lot of Merge commits) like this one.

We will do squash merge this time :)

**Comment by YinYangOfDao** [comment]
> I think this will create a real k8s cluster, right? jobmanager will need a real k8s cluster to have some unit tests, do not need a gpu worker though. How can I add that in this CI pipeline?

Currently I think I use CPU machines as default resource type for CI test, not GPU required.

**Comment by xudifsd** [comment]
> > I think this will create a real k8s cluster, right? jobmanager will need a real k8s cluster to have some unit tests, do not need a gpu worker though. How can I add that in this CI pipeline?
> 
> Currently I think I use CPU machines as default resource type for CI test, not GPU required.

Yes, I think test in jobmanager also do not need gpu worker.

File: src/ClusterBootstrap/params.py [hunk 1]
```diff
@@ -664,14 +664,15 @@
...
+        "runscriptonroles infra worker ./scripts/disable_kernel_auto_updates.sh",
+        "runscriptonroles infra worker ./scripts/docker_network_gc_setup.sh",
         "genscripts",
         "runscriptonroles infra worker ./scripts/dns.sh",
         "-y deploy",
         "-y updateworker",
         "-y kubernetes labels",
         "-y gpulabel",
         "kubernetes start nvidia-device-plugin",
-        "kubernetes start flexvolume",
```
Comment (xudifsd) [inline src/ClusterBootstrap/params.py]: Why remove this?

Comment (YinYangOfDao) [reply src/ClusterBootstrap/params.py]: I didn't remember removing it...maybe it's not synced. I don't know when is this added to the pipeline.

**Pull request merged**